client/CMessage.h

```diff
@@ -21,10 +21,10 @@ class CMessage
 {
 public:
 	//Function usd only in CMessage.cpp
-	static std::pair<int,int> getMaxSizes(std::vector<std::vector<SDL_Surface*> > * txtg, int fontHeight);
+	static std::pair<int, int> getMaxSizes(std::vector<std::vector<SDL_Surface *>> * txtg, int fontHeight);

```

Comment:
I think I reported this case already, but this change looks really dumb imo with that pointer-spacebar mangling.